hal/src/thumbv6m/sercom/v1/uart.rs

```diff
@@ -253,10 +259,17 @@ macro_rules! uart {
 
                             let (rxpo, txpo) = padout.rxpo_txpo();
                             w.rxpo().bits(rxpo);
+                            #[cfg(feature = "samd20")]
+                            w.txpo().bit(txpo != 0);
+                            #[cfg(not(feature = "samd20"))]
                             w.txpo().bits(txpo);
 
                             w.form().bits(0x00);
-                            w.sampr().bits(0x00); // 16x oversample fractional
+
+                            // 16x oversample fractional
+                            #[cfg(not(feature = "samd20"))]
+                            w.sampr().bits(0x00);
```

Comment:
does SAMD20 not have oversampling setting? If it does and it's just different, make sure to still set it